### PR TITLE
Feature/sc 158400 add user credentials preset

### PR DESCRIPTION
**Pull request by alexbourret**
Story details: https://app.shortcut.com/dataiku/story/158400

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 - Add filter field
 - Convert OData dates to DSS format
+- Add individual user credentials preset
```
Comment (MayeulRousselet) [inline CHANGELOG.md:7]: and renamed SAP/ODATA login parameter to Service account

File: parameter-sets/user-accounts/parameter-set.json [hunk 2]
```diff
@@ -0,0 +1,65 @@
+{
+    "meta": {
+        "label": "User accounts",
```
Comment (MayeulRousselet) [inline parameter-sets/user-accounts/parameter-set.json:3]: I'll rename it to Personal user accounts, to have the same term as in the Custom Dataset UI

**Review by MayeulRousselet — CHANGES_REQUESTED** [review]
On a general note, is it possible to change the Parameter set ID of the user account to basic-auth? to Be consistent with pi-system:
Capture d’écran 2023-11-13 à 9 36 27 AM

File: parameter-sets/user-accounts/parameter-set.json (hunk 2 above)
Comment (alexbourret) [reply parameter-sets/user-accounts/parameter-set.json]: 👍 done in [aaed917](https://github.com/dataiku/dss-plugin-sap-odata/pull/9/commits/aaed917b0a321eee724651010f752fddf7f088c4)

File: CHANGELOG.md (hunk 1 above)
Comment (alexbourret) [reply CHANGELOG.md:7]: done in [aaed917](https://github.com/dataiku/dss-plugin-sap-odata/pull/9/commits/aaed917b0a321eee724651010f752fddf7f088c4)

**Comment by alexbourret** [comment]
The parameter set id was changed to be in line with pi-system plugin
Screenshot 2023-11-17 at 14 21 10

File: CHANGELOG.md (hunk 1 above)
Comment (MayeulRousselet) [reply CHANGELOG.md:7]: I disagree your honor

Comment (MayeulRousselet) [reply CHANGELOG.md:7]: The changelog should contains 
```suggestion
- Add individual user credentials preset and renamed SAP/ODATA login parameter to Service account
```

**Review by MayeulRousselet — CHANGES_REQUESTED** [review]
Cf my coomments, just change the changelog

File: CHANGELOG.md (hunk 1 above)
Comment (alexbourret) [reply CHANGELOG.md:7]: oops [b3909c3](https://github.com/dataiku/dss-plugin-sap-odata/pull/9/commits/b3909c3501237807c0006d0a3756f8d686154133)

**Review by MayeulRousselet — APPROVED** [review]

**Pull request merged**